README.md

```diff
@@ -201,6 +201,17 @@ Rationale: some labels are not very selective, or very rarely come up in queries
 -	Mentor(s): Ganesh Vernekar (@codesome)
 -	Issue: https://github.com/prometheus/prometheus/issues/21.
 
+#### Allow running prombench locally
+-	Description: Currently prombench development requires the user to setup GKE environment. Having prombench to run in a local k8s cluster would make adding changes much easier and faster. Support for kubeconfig and a proper way to switch local and GKE environment is not available yet. Adding these features will be essential to the completion of the project.
+-	Recommended Skills: CI, Golang, Kubernetes
+-	Mentor(s): Krasi Georgiev (@krasi-georgiev), Hrishikesh Barman (@geekodour)
```

Comment:
can you add an issue here?

Reply:
sure, I'll create and add the issue here.

Reply:
Added.